src/app/shared/editor/editor.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { MonacoEditorModule } from "ngx-monaco-editor-v2";
+
+import { SqlEditorComponent } from "./components/sql-editor/sql-editor.component";
+import { YamlEditorComponent } from "./components/yaml-editor/yaml-editor.component";
+import { FormsModule } from "@angular/forms";
+
+@NgModule({
+    declarations: [SqlEditorComponent, YamlEditorComponent],
+    imports: [CommonModule, FormsModule, MonacoEditorModule, MonacoEditorModule.forRoot()],
```

Comment:
`MonacoEditorModule, MonacoEditorModule.forRoot()` - looks weird, I guess 1 should be enough.

Also, please remove `MonacoEditorModule` from the main `AppModule`

Reply:
Fxied